images/env-android/Dockerfile

```diff
@@ -30,7 +30,8 @@ RUN apt-get install -y \
     intltool \
     libtool \
     make \
-    pkg-config
+    pkg-config \
+    util-linux
```

Comment:
I don't think we need it in env-android. This will install util-linux for host system, but not for the target system (android).

If libuuid is not part of Android NDK (i.e. is not available out of the box in android toolchain), I guess we can just disable it in android build.